extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_linkAuthentication.php

```diff
@@ -1,6 +1,6 @@
-<?php if ( isset( $model['subtitle'] ) ): ?>
+<?php if ( $model['title']['key'] === 'global-navigation-anon-register' ): ?>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)